src/Toolkit/V.php

```diff
@@ -390,6 +390,23 @@ public static function __callStatic(string $method, array $arguments): bool
         return true;
     },
 
+    /**
+     * Checks for empty values
+     */
+    'empty' => function ($value = null): bool {
+        $empty = ['', null, []];
+
+        if (in_array($value, $empty, true) === true) {
+            return true;
+        }
```

Comment:
What about empty space, `'    '`? Do you think that we should use `trim`?

Reply:
Maybe trimming white space would make sense, I agree

Reply:
I don't think the validator should do that. It might be the right choice 90% of the time, but a single space could also be interpreted as an actual choice by the user to have that as a intended value.

Reply:
Ok